wordpress-mobile/WordPress-Android
# Fixed issue with sharing external media/links from other apps

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/10570

Following gb-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1817

To test:

**Test sharing media (Image/Video)**
1. Make sure that default Editor is Gutenberg
2. Go to Photos and Videos
3. Find proper image/video
4. Click on the share button and choose WPAndroid
5. WPAndroid will open and image/video should be uploaded with success

**Test sharing link from Web Browser in Gutenberg**
1. Make sure that default Editor is Gutenberg
2. Open some Web Browser
3. Visit random url
4. Click on the share button and choose WPAndroid
5. WPAndroid will open and the title with a clickable link from Web Browser should be in the Post.

**Test sharing link from Web Browser in Aztec**
1. Make sure that default Editor is Aztec
2. Open some Web Browser
3. Visit random url
4. Click on the share button and choose WPAndroid
5. WPAndroid will open and the title with a clickable link from Web Browser should be in the Post.

**Test sharing link from other apps (Youtube, Twitter...) in Gutenberg**
1. Make sure that default Editor is Gutenberg
2. Open another app (Youtube, Twitter...)
3. Pick some video/Twitter post ....
4. Click on the share button and choose WPAndroid
5. WPAndroid will open with a link in the Post.

**Test sharing link from other apps (Youtube, Twitter...) in Aztec**
1. Make sure that default Editor is Aztec
2. Open another app (Youtube, Twitter...)
3. Pick some video/Twitter post ....
4. Click on the share button and choose WPAndroid
5. WPAndroid will open with a link in the Post.

PR submission checklist:
